client/src/pages/ProfileList/List.tsx

```diff
@@ -89,47 +107,31 @@ export default function List(): JSX.Element {
             </Box>
           </Grid>
         </Grid>
-        <Grid container justify="center" spacing={6} className={`${classes.root}`}>
-          {profiles.map((profile, key) => {
-            const { firstName, lastName, user, description } = profile;
-            return (
-              user && (
-                <Grid item key={user}>
-                  <Card style={{ height: '100%' }} variant="outlined" key={key}>
-                    <CardActionArea component={Link} to={`profile/${user}`}>
-                      <Grid
-                        justify="center"
-                        alignItems="center"
-                        direction="column"
-                        container
-                        className={`${classes.card}`}
-                      >
-                        <Avatar src={profilePhoto} className={classes.avatar} />
-
-                        <CardContent>
-                          <Typography className={classes.fullName} variant="h6">
-                            {firstName} {lastName}
-                          </Typography>
-                          <Rating name="read-only" value={placeholder.rating} readOnly />
+        <Grid container spacing={10} className={`${classes.root}`} sm>
+          {profiles
+            .filter((profile: Profile) => {
+              if (profile.address && profile.address.includes(searchTerm)) {
+                return profile;
+              }
+            })
+            .filter((profile: Profile) => {
+              //TODO: Implement date search
+              // const requestDays = moment.range(new Date(selection.startDate), new Date(selection.endDate));
 
-                          <Typography>{description ? description : defaultDescription}</Typography>
-                        </CardContent>
-                      </Grid>
-                    </CardActionArea>
-                    <Divider light />
-                    <Grid container className={`${classes.bottom}`} spacing={2}>
-                      <Grid item>
-                        <Typography className={classes.location}>{placeholder.location}</Typography>
-                      </Grid>
-                      <Grid item>
-                        <Typography className={classes.price}>${placeholder.price}/hr</Typography>
-                      </Grid>
-                    </Grid>
-                  </Card>
-                </Grid>
-              )
-            );
-          })}
+              // const days = Array.from(requestDays.by('day'));
+              // days.map((day) => {
+              //   if (day) {
+              //     const weekday = moment(day).format('dddd');
+              //     if (availability[weekday as key] === false) {
+              //       return;
+              //     }
+              //   }
+              // });
+              return profile;
+            })
+            .map((profile, key) => (
+              <UserCard profile={profile} key={key} />
```

Comment:
This is the correct use of the `key` prop 🙌🏼 

As an aside, I generally recommend passing props explicitly (e.g. each field of profile as a single prop rather than the profile object itself). No changes required though 👍🏼

Reply:
`          {profiles
            .filter((profile: Profile) => {
              return locationFilter(dateFilter(profile));
            })
            .map((profile, key) => (
              <UserCard profile={profile} key={key} />
            ))}`